frontend/src/queries/example.ts

```diff
@@ -0,0 +1,62 @@
+import { query2dcids, Query, CategoryType } from "./query"
+import { Dcid } from "./dcids"
+
+import { DCIDS } from "./example_dcids"
+
+
+const Categories = {
+    'gender': new Set<string>([
+        'Male',
+        'Female',
+    ]),
+    'ethnicity': new Set<string>([
+        'HispanicOrLatino',
+        'OtherPacificIslander',
+        'AmericanIndianOrAlaskaNative',
+        'Asian',
+        'Black',
+        'TwoOrMoreRaces',
+        'WhiteAlone',
+    ]),
+    'occupation': new Set<string>([
+        'SOCComputerMathematicalOccupation',
+        'SOCEngineersOccupation',
+        'SOCLifePhysicalSocialScienceOccupation',
+        'SOCLifeScientistsOccupation',
+        'SOCMathematicalScienceOccupation',
+        'SOCPhysicalScientistsOccupation',
+        'SOCPsychologistsOccupation',
+        'SOCSocialScientistsRelatedWorkersOccupation',
+        'ScienceAndEngineeringOccupation',
+        'ScienceAndEngineeringRelatedOccupation',
+        'NonScienceAndEngineeringOccupation',
+    ]),
+}
+
+// Dimensions to Category mapping.
+const Dim2Cat = {}
+// console.log(Categories)
+
+Object.keys(Categories).forEach((key) => {
+    console.log(key)
```

Comment:
Consider removing log statement.

Reply:
This PR is just a demo; I wasn't planning on merging. I was planning to write tests for querying confirmed datasets once we have a list of DCIDs that are ready for publishing.